ga4gh/avrotools.py

```diff
@@ -14,6 +14,68 @@
 LONG_MAX_VALUE = (1 << 63) - 1
 
 
+class CreationToolException(Exception):
+    """
+    Something went wrong in the CreationTool
+    """
+
+
+class CreationTool(object):
+    """
+    Provides methods for creating and populating objects with data
+    """
```

Comment:
Maybe we could make this a real class with some state, rather than collection of class methods? Something like 'InstanceGenerator`, which takes a class/schema as an argument at instantiation time, and then has methods like `getTypicalInstance` or `getRandomInstance`. I'm not hugely keen on the classmethod approach --- as @macieksmuga pointed out somewhere else, it's very.... Java.